src/data-access/getFolders.js

```diff
@@ -0,0 +1,14 @@
+import { BASE_URL } from "./BASE_URL";
+
+export const getFolders = async () => {
+  const response = await fetch(`${BASE_URL}users/1/links`);
+  const result = await response.json();
+  return result;
+};
+
+export const getFolder = async ({ folderId }) => {
+  const query = `?folderId=${folderId}`;
+  const response = await fetch(`${BASE_URL}users/1/links${query}`);
+  const result = await response.json();
+  return result;
+};
```

Comment:
P3:
반복되는 부분들이 있어서 getFolder, getFolders로 데이터 패치 함수를 나누지 말고 한 함수안에서 folderId 여부로 불러오게 변경해도 될 것 같습니다.

```suggestion
export const getFolders = async ({folderId}) => {
  const query = folderId ? `?folderId=${folderId}` : "";
  
  const response = await fetch(`${BASE_URL}users/1/links${query}`);
  const result = await response.json();
  
  return result;
}```